src/mmda/predictors/d2_predictors/bibentry_detection_predictor.py

```diff
@@ -120,13 +119,13 @@ def postprocess(self,
 
             current_id = next(id_counter)
 
-            original_box_groups.append(
-                BoxGroup(
-                    boxes=[model_output_box],
-                    id=current_id,
-                    type="raw_model_prediction"
-                )
+            box_group = BoxGroup(
+                boxes=[model_output_box],
+                id=current_id
             )
+            box_group.metadata.set('type', 'raw_model_prediction')
```

Comment:
was this change related to a breaking change in how metadata and type are defined for BoxGroup? or do both ways of setting "type" work and this one is more forward-looking for how we want to define metadata from now on?

Reply:
The first - this is related to a breaking change in MMDA, BoxGroup no longer has "type" directly on it, it has to be part of metadata

Reply:
Gotcha, I take it this is one of the grab bag of changes you get to pick up with a new version whether you want them all or not!